### PR TITLE
Get/api/articles 8

### PR DESCRIPTION
**Pull request by apwb89**

### CODE REVIEW

**Comment by apwb89** [comment]
Might have just broken something in app.test.js when trying to resolve

**Comment by apwb89** [comment]
Think i worked out the conflicts?

File: view-db.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+\c nc_news_test
+
+SELECT * FROM articles;
+SELECT * FROM comments;
+SELECT * FROM topics;
+SELECT * FROM users;
+
```
Comment (cavalierswain) [inline view-db.sql:5]: I like that you're visualising the db! It really helps!

**Comment by cavalierswain** [comment]
Looks like it you sorted it! 

Good work!

**Pull request merged**